ProjFS.Mac/PrjFSKextTests/MockProc.cpp

```diff
@@ -0,0 +1,23 @@
+#include "MockProc.hpp"
+#include <string.h>
+
+int proc_pid(proc_t)
+{
+    return 1;
+}
+
+char* return_proc_name;
+
+void proc_name(int pid, char * buf, int size)
+{
+    if (return_proc_name != nullptr)
+    {
+        strcpy(buf, return_proc_name);
```

Comment:
Please always use `strlcpy`, or in some exceptional circumstances `strncpy`. `strcpy` is almost never appropriate, as it doesn't check for enough space in the destination.

[The real `proc_name` implementation](https://github.com/apple/darwin-xnu/blob/a449c6a3b8014d9406c2ddbdc81795da24aa7443/bsd/kern/kern_proc.c#L874) uses `strlcpy`, so let's definitely use that please.